### DISCUSSION TITLE
Can I create a navigation menu with some categories I want?

### DISCUSSION THREAD

**Discussion by SuyeonChoi** [mmistakes/minimal-mistakes#2584] [Q&A] [answered]

**Comment by iBug** [comment, marked as answer]
This is only possible with [Jekyll Paginate V2][1], which means you'll have to build your site using some kind of CI service instead of vanilla GitHub Pages. The implementation is relatively easy, however:

<https://github.com/iBug/iBug-source/blob/63f7c7d9e714e8b94dad5098a83616fc5b04c85c/blog/index.html#L10-L12>

  [1]: https://github.com/sverrirs/jekyll-paginate-v2
